StudentDriver/StudentDriver/Models/User.cs

```diff
@@ -12,7 +12,7 @@ public class User
         public string Email { get; set; }
         public string FirstName { get; set; }
         public string LastName { get; set; }
-        public int DrivingSchoolId { get; set; }
+        public string DrivingSchoolId { get; set; }
```

Comment:
Are we using unique id's? I thought we were using mongo id's for the time being?